ui/js/controllers/perf/graphs.js

```diff
@@ -1054,7 +1054,8 @@ perf.controller('TestChooserCtrl', function($scope, $uibModalInstance, $http,
                       subtests: $scope.includeSubtests ? 1 : 0 }).then(function(seriesList) {
                           $scope.unselectedTestList = _.sortBy(
                               _.filter(seriesList,
-                                       { platform: $scope.selectedPlatform }), 'name');
+                                       { platform: $scope.selectedPlatform,
+                                         hasSubtests: true }), 'name');
```

Comment:
I did console.log(seriesList), and it was a list of objects with ``hasSubtests`` property.

I think line 50 on the same file parses ``has_subtests`` to the property ``hasSubtests``, or did I not understand what you mean?